circle.yml

```diff
@@ -0,0 +1,14 @@
+machine:
+  pre:
+    - wget "http://ftp.debian.org/debian/pool/main/s/shellcheck/shellcheck_0.4.4-4_amd64.deb"
+    - sudo dpkg -i "shellcheck_0.4.4-4_amd64.deb"
+    - export PATH="/usr/local/bin:$PATH"
+    - git clone https://github.com/sstephenson/bats.git /tmp/bats && pushd /tmp/bats && sudo ./install.sh /usr/local && popd
+
+test:
+  override:
+    - curl -s https://raw.githubusercontent.com/18F/laptop/master/seekrets-install | bash -
```

Comment:
Since we have the repo checked out here for CI, should be able to just `cat seekrets-install | bash -` instead of using curl to grab it from github again